client/my-sites/stats/geochart/index.jsx

```diff
@@ -50,75 +64,99 @@ export default React.createClass( {
 			this.resize.cancel();
 		}
 		window.removeEventListener( 'resize', this.resize );
-	},
+	}
 
-	recordEvent() {
+	recordEvent = () => {
 		analytics.ga.recordEvent( 'Stats', 'Clicked Country on Map' );
```

Comment:
Since we're refactoring this component, could we update this to use `recordGoogleEvent` from `state/analytics/actions`